ocean_lib/ocean/test/test_ocean_compute.py

```diff
@@ -19,8 +18,7 @@
 
 def test_build_cluster_attributes():
     data_provider = DataServiceProvider()
-    config = Ocean.config
-    compute = OceanCompute(config=config, data_provider=data_provider)
+    compute = OceanCompute(config={}, data_provider=data_provider)
```

Comment:
Instead of changing this to `{}`, pass in the `config` fixture like so. Do this same change for `test_build_container_attributes` and `test_build_server_attributes`

```python
def test_build_cluster_attributes(config):
    data_provider = DataServiceProvider()
    compute = OceanCompute(config=config, data_provider=data_provider)
...
```